Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_CPPException.mm

```diff
@@ -92,6 +92,47 @@ void __cxa_throw(void *thrown_exception, std::type_info *tinfo,
 }
 }
 
+static void handleNSException(NSException *exception) {
```

Comment:
this mostly originates from the handler code in BSG_KSCrashSentry_NSException, with a few adaptations